### PR TITLE
UP-2823 - added helpful text on registration page for both the drop d…

### PR DESCRIPTION
**Pull request by stalele**
…own boxes.

##### Checklist

-   [ ] the [individual contributor license agreement][] is signed
-   [x] commit message follows [commit guidelines][]
-   [x] [message properties][] have been updated with new phrases
-   [x] view conforms with [WCAG 2.0 AA][]

##### Description of change

added helpful text on registration page for both the drop down boxes

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Comment by drewwills** [comment]
@ChristianMurphy and @cousquer -- Is there anything we should be adding to this change to meet our accessibility goals?  Perhaps a `<label>` tag?

File: uportal-war/src/main/resources/properties/i18n/Messages.properties [hunk 1]
```diff
@@ -552,6 +552,8 @@ rating.saved.unsuccessfully=Your rating did not save.  Please try again!
 rating.retrieved.unsuccessfully=Sorry, your previous rating of the portlet could not be retrieved
 recently.selected=Recently Selected
 register.new.portlet=Register New Portlet
+application.information = WAR file name
+portlet.information= Portlet Name
```
Comment (ChristianMurphy) [inline uportal-war/src/main/resources/properties/i18n/Messages.properties]: Could you put these strings into Google translate and add the translations to `Messages_fr.properties` 🇫🇷  and `Messages_es.properties` 🇪🇸 ?

File: uportal-war/src/main/webapp/WEB-INF/flows/edit-portlet/choosePortlet.jsp [hunk 2]
```diff
@@ -77,7 +77,7 @@ PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
             </div>
             <!-- Application Dropdown -->
             <div class="form-group">
-                <div class=" col-md-3">
+                <div class=" col-md-3"> <spring:message code="application.information"/>
```
Comment (ChristianMurphy) [inline uportal-war/src/main/webapp/WEB-INF/flows/edit-portlet/choosePortlet.jsp]: Could you wrap the `<spring:message>` and the `<select>` in a `<label>`? That would improve the accessibility for this! 😄

File: uportal-war/src/main/webapp/WEB-INF/flows/edit-portlet/choosePortlet.jsp [hunk 3]
```diff
@@ -87,7 +87,7 @@ PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
             </div>
             <!-- Portlet Dropdown -->
             <div class="form-group">
-                <div class="col-md-3">
+                <div class="col-md-3"><spring:message code="portlet.information"/>
```
Comment (ChristianMurphy) [inline uportal-war/src/main/webapp/WEB-INF/flows/edit-portlet/choosePortlet.jsp]: Could you wrap the `<spring:message>` and the `<select>` in a `<label>`? That would improve the accessibility for this! 😄

**Comment by stalele** [comment]
Added changes as suggested in PR

File: uportal-war/src/main/resources/properties/i18n/Messages.properties (hunk 1 above)
Comment (ChristianMurphy) [inline uportal-war/src/main/resources/properties/i18n/Messages.properties]: added in 8e08b9d

**Pull request merged**